docs/tests-github.md

```diff
@@ -0,0 +1,55 @@
+---
+category: DevelopInDepth
+
+---
+# Using GitHub Actions
+
+[Github Action](https://github.com/features/actions) makes it easy to automate all your software workflows, now with world-class CI/CD. Build, test, and deploy your code right from GitHub. Make code reviews, branch management, and issue triaging work the way you want.
```

Comment:
This is still the claim copied from `https://github.com/features/actions`. I'm not sure if copying any content from other sites might cause legal problems as we claim the copyright in our license file. (ping @tsteur ) So we either need to mention where the text is copied from (if that is allowed at all) or use another text here that describes what we are using GitHub Actions for maybe.

Reply:
```suggestion
[Github Action](https://github.com/features/actions) are used to automate our software workflows including running javascript builds, test suites, releases and code checks.
```